airflow/providers/salesforce/hooks/salesforce.py

```diff
@@ -146,14 +155,17 @@ def get_object_from_salesforce(self, obj: str, fields: Iterable[str]) -> dict:
         :return: all instances of the object from Salesforce.
         :rtype: dict
         """
+        if not fields:
+            fields = self.get_available_fields(obj)
```

Comment:
hmm... so you make it optional, but if not supplied, it will fail if not object...

starting to wonder if this thould be a subclass `SalesforceObjectToGcsOperator` instead of parameter